### PR TITLE
new icon: subversion (original, original-wordmark)

### PR DESCRIPTION
**Pull request by MichielPater**
And re-sorted icons starting with 's' alphabetically

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -3511,34 +3541,22 @@
...
                 "original",
-                "original-wordmark",
-                "plain",
-                "plain-wordmark"
+                "original-wordmark"
             ],
             "font": [
-                "plain",
-                "plain-wordmark"
+                "original"
```
Comment (Panquesito7) [inline devicon.json:3556]:
```suggestion
                "original",
                "original-wordmark"
```

File: devicon.json [hunk 2]
```diff
@@ -3511,34 +3541,22 @@
...
-    {
-        "name": "spss",
-        "tags": ["package", "statistics"],
-        "versions": {
-          "svg": ["original", "plain"],
-          "font": ["plain"]
-        },
-        "color": "#cc1e4c",
+        "color": "#809cc8",
         "aliases": []
```
Comment (Panquesito7) [inline devicon.json:3560]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
            {
                "base": "original-wordmark",
                "alias": "original-wordmark"
            }
        ]
```

File: icons/subversion/subversion-original-wordmark.svg [hunk 3]
```diff
@@ -0,0 +1 @@
....484-.816 6.852-1.508 10.105-2.074zm23.922 1.258c-3.152.438-5.98.957-8.469 1.539-2.848.672-5.141 1.352-6.875 2.043-1.738.676-2.543 1.227-2.43 1.645.102.375.828.547 2.184.508.699-.004 1.828-.078 3.395-.207l6.074-.566c2.457-.238 4.492-.398 6.121-.488zm-7.004 7.461c-.695.008-1.656.059-2.887.145l-4.59.387-7.078.656-2.363.148v4.77c3.727-.508 7.098-1.121 10.102-1.824 2.953-.695 5.223-1.371 6.816-2.027s2.344-1.207 2.219-1.66c-.102-.387-.852-.582-2.219-.594zm7.004 1.676c-3.273 1.203-7.336 2.383-12.195 3.523C7.797 70.465 3.891 71.23.59 71.699v.039h23.922zm-22.922 1c.012-.004.02 0 .031 0 .063 0 .129.02.172.066.063.059.078.16.043.238-.043.09-.152.141-.254.125-.117-.016-.207-.121-.195-.23.004-.102.094-.187.203-.199zm.848 0c.051 0 .102.012.141.051.078.055.105.16.07.242-.035.09-.129.152-.23.145-.086-.004-.168-.055-.203-.125-.047-.098-.008-.23.094-.281a.28.28 0 0 1 .129-.031zm-.816 1.406a.23.23 0 0 1 .223.152c.031.094-.02.219-.121.258-.098.051-.23.02-.293-.07s-.059-.215.031-.285c.043-.035.102-.059.16-.055zm.816 0c.078-.004.148.039.191.102.063.094.039.227-.059.285-.094.07-.246.059-.32-.031a.22.22 0 0 1 .023-.312c.043-.027.105-.047.164-.043zm0 0" fill="rgb(50.196078%,61.176471%,78.431373%)"/></svg>
```
Comment (Panquesito7) [inline icons/subversion/subversion-original-wordmark.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" xmlns:v="https://vecta.io/nano"><path d="M.59 55.484h126.82v16.258H.59zm0 0" fill-rule="evenodd" fill="rgb(100%,100%,100%)"/><path d="M30.238 60.363c-.5 0-.937.063-1.309.199-.359.145-.641.34-.84.59-.191.266-.289.559-.289.883 0 .352.094.637.281.875s.434.418.742.551c.316.133.707.25 1.176.355l.828.207c.199.063.371.16.504.285s.195.281.195.484c0 .254-.117.457-.363.625s-.551.258-.941.258c-.289 0-.516-.039-.687-.125-.18-.086-.312-.184-.402-.32-.098-.125-.195-.289-.277-.477-.062-.164-.141-.289-.242-.363-.09-.086-.203-.129-.348-.129-.172 0-.309.059-.418.164a.5.5 0 0 0-.168.375c0 .254.094.508.277.785a2.06 2.06 0 0 0 .727.652c.406.215.926.324 1.566.324.531 0 .992-.074 1.371-.242.398-.16.688-.391.898-.684l.023-.055.043-.051.086-.172.027-.066.02-.055.016-.055.039-.129.008-.059.02-.07.008-.066v-.059l.004-.066v-.078c0-.297-.059-.551-.168-.762a1.49 1.49 0 0 0-.492-.52c-.207-.133-.457-.246-.746-.344-.297-.098-.625-.184-.996-.266l-.617-.148a3.5 3.5 0 0 1-.371-.148.84.84 0 0 1-.301-.234c-.074-.086-.109-.187-.109-.301 0-.195.109-.367.316-.504.207-.145.48-.203.813-.203.371 0 .637.055.809.191.164.125.297.297.426.523.09.164.168.27.242.34s.188.102.344.102.297-.059.414-.172c.109-.105.164-.242.164-.383 0-.16-.047-.32-.137-.484s-.223-.332-.422-.48c-.184-.148-.426-.277-.715-.371s-.633-.129-1.027-.129zm8.422 0c-.187 0-.352.051-.469.18s-.184.305-.184.555v2.887c0 .449.059.84.168 1.164.102.32.266.594.492.801s.52.371.883.465c.352.098.781.148 1.27.148.426 0 .789-.043 1.105-.129.305-.094.57-.25.805-.441a1.77 1.77 0 0 0 .566-.828c.102-.316.141-.707.141-1.172v-2.895c0-.25-.055-.437-.18-.555-.117-.129-.281-.18-.48-.18s-.352.051-.469.18c-.117.117-.176.305-.176.555v2.984c0 .512-.098.898-.316 1.148-.207.25-.574.375-1.09.375-.359 0-.648-.059-.855-.187a.94.94 0 0 1-.422-.523c-.082-.215-.129-.5-.129-.84v-2.957c0-.25-.059-.43-.168-.555-.125-.129-.293-.18-.492-.18zm11.051.098c-.289 0-.488.059-.613.176s-.184.305-.184.57v4.508c0 .258.059.445.184.57.117.121.324.176.613.176h1.965c.344 0 .656-.016.906-.047a2.52 2.52 0 0 0 .676-.164 1.62 1.62 0 0 0 .75-.594c.191-.266.277-.57.277-.902 0-.73-.422-1.223-1.273-1.469.652-.289.973-.723.973-1.32a1.28 1.28 0 0 0-.098-.5 1.39 1.39 0 0 0-.266-.426c-.121-.137-.254-.238-.406-.32-.18-.102-.379-.168-.605-.199-.227-.043-.5-.059-.809-.059zm.512.906h1.07c.469 0 .828.051 1.063.148.223.102.344.305.344.605a.65.65 0 0 1-.109.379.67.67 0 0 1-.402.336c-.184.063-.437.086-.762.086h-1.203zm0 2.422h1.363c.461 0 .805.066 1.031.207.227.133.344.363.344.66 0 .598-.441.902-1.328.902h-1.41zm0 0" fill="rgb(50.196078%,61.176471%,78.431373%)"/><path d="M59.684 60.363c-.184 0-.336.051-.453.164a.52.52 0 0 0-.18.383.88.88 0 0 0 .027.199c.016.078.047.152.07.219l.082.215 1.535 3.891.168.422a2.01 2.01 0 0 0 .195.363c.082.102.18.184.297.25.125.066.27.094.449.094.172 0 .316-.027.445-.094.117-.066.215-.148.297-.25.07-.102.133-.23.188-.363l.168-.422 1.563-3.922.078-.215c.027-.062.051-.133.063-.199.016-.074.039-.137.039-.191 0-.094-.039-.18-.094-.266s-.125-.152-.215-.207a.72.72 0 0 0-.316-.07c-.152 0-.273.031-.363.098a.76.76 0 0 0-.187.223c-.039.078-.09.227-.168.445l-1.469 4.055-1.461-4.031a2.94 2.94 0 0 0-.281-.609c-.09-.129-.254-.18-.477-.18zm33.059 0c-.5 0-.934.063-1.293.199-.371.145-.648.34-.848.59-.191.266-.289.559-.289.883 0 .352.102.637.281.875s.43.418.746.551c.309.133.695.25 1.164.355l.836.207c.207.063.379.16.504.285s.203.281.203.484c0 .254-.129.457-.363.625-.246.168-.562.258-.949.258-.289 0-.516-.039-.687-.125s-.309-.184-.41-.32c-.098-.125-.176-.289-.254-.477-.074-.164-.16-.289-.25-.363a.52.52 0 0 0-.352-.129c-.172 0-.312.059-.422.164s-.164.227-.164.375c0 .254.09.508.27.785.184.27.426.484.715.652.414.215.941.324 1.582.324.531 0 .988-.074 1.379-.242.387-.16.684-.391.883-.684l.039-.055.027-.051.039-.062.055-.109.027-.066.012-.055.027-.055.02-.066.008-.062.02-.059.008-.07.012-.066v-.059l.008-.066v-.078c0-.297-.059-.551-.168-.762-.117-.207-.285-.383-.492-.52s-.449-.246-.746-.344-.629-.184-.988-.266l-.633-.148-.371-.148a.86.86 0 0 1-.297-.234c-.062-.086-.098-.187-.098-.301 0-.195.102-.367.309-.504.207-.145.477-.203.82-.203.359 0 .633.055.793.191.164.125.309.297.426.523.09.164.172.27.254.34.074.07.191.102.336.102a.54.54 0 0 0 .406-.172c.117-.105.172-.242.172-.383a1.01 1.01 0 0 0-.141-.484 1.4 1.4 0 0 0-.41-.48c-.187-.148-.437-.277-.727-.371s-.633-.129-1.027-.129zm8.469 0c-.187 0-.344.051-.461.18-.129.125-.184.305-.184.555v4.727c0 .246.063.426.184.551s.273.188.461.188.359-.055.484-.176c.117-.125.176-.309.176-.562v-4.727c0-.25-.059-.437-.176-.555-.125-.129-.293-.18-.484-.18zm9.031 0c-.488 0-.93.063-1.32.219-.387.133-.715.344-.988.609a2.74 2.74 0 0 0-.617.984c-.137.391-.211.813-.211 1.27s.063.871.195 1.258.344.719.605.996c.273.277.605.484 1.004.641.398.148.852.223 1.359.223a3.8 3.8 0 0 0 1.352-.223c.391-.141.727-.355.996-.633.273-.285.469-.609.605-.988s.203-.809.203-1.266c0-.629-.125-1.172-.371-1.645a2.53 2.53 0 0 0-1.09-1.07l-.086-.043-.094-.039-.102-.047-.094-.043-.102-.031-.102-.023-.109-.039-.098-.023-.109-.016-.113-.02-.117-.02-.113-.008-.113-.02h-.121l-.129-.004zm9.16 0c-.156 0-.293.027-.418.109-.137.078-.23.184-.285.305-.055.109-.066.266-.066.465v4.625c0 .227.051.402.168.52.109.121.266.176.445.176s.328-.055.445-.176.168-.285.168-.52v-3.523l2.469 3.504.207.277c.074.094.148.176.23.242s.172.117.262.148a1.1 1.1 0 0 0 .316.047c.469 0 .707-.258.707-.789v-4.715c0-.238-.055-.418-.164-.531s-.254-.164-.441-.164c-.18 0-.324.051-.43.164s-.164.293-.164.531v3.566l-2.523-3.539-.207-.301-.195-.238a.75.75 0 0 0-.223-.133.77.77 0 0 0-.301-.051zm-48.969.098a1.14 1.14 0 0 0-.32.074c-.129.059-.211.137-.266.242-.062.109-.086.254-.086.43v4.508c0 .258.063.453.188.57s.34.176.621.176h3.566c.199 0 .348-.051.457-.137s.152-.199.152-.34c0-.148-.043-.258-.152-.352-.109-.082-.258-.125-.457-.125h-3.055v-1.719h2.711c.207 0 .359-.043.449-.121a.43.43 0 0 0 .141-.336c0-.141-.039-.25-.141-.336s-.25-.125-.449-.125h-2.711v-1.477h2.945c.219 0 .355-.043.465-.125.098-.086.156-.203.156-.344s-.059-.246-.156-.336c-.109-.09-.246-.129-.465-.129H70.57c-.047 0-.094-.004-.137 0zm10.215 0c-.289 0-.492.059-.617.176s-.191.305-.191.57v4.617c0 .246.059.426.176.551s.277.188.484.188c.191 0 .355-.055.473-.176.125-.125.18-.309.18-.562v-1.93h.457c.227 0 .426.027.582.09s.297.16.449.316a4.97 4.97 0 0 1 .512.676l.52.816.289.434a1.15 1.15 0 0 0 .254.242c.109.066.223.094.379.094a.76.76 0 0 0 .328-.066c.09-.047.16-.098.215-.184a.39.39 0 0 0 .082-.234c0-.082-.047-.219-.137-.422s-.227-.426-.391-.68-.348-.492-.562-.719c-.207-.227-.434-.387-.66-.512.551-.109.953-.293 1.223-.566.281-.262.41-.625.41-1.066 0-.219-.039-.43-.121-.621a1.34 1.34 0 0 0-.348-.512c-.152-.152-.336-.266-.547-.344a2.2 2.2 0 0 0-.559-.145l-.746-.031zm29.594.855c.371 0 .695.086.977.254.289.156.5.398.645.727l.027.055.027.074.027.063.027.059.02.07.02.059.016.086.012.063.012.07.016.078.008.078.004.078.016.07v.086l.008.07v.09c0 .477-.086.879-.258 1.207a1.69 1.69 0 0 1-.66.719c-.289.156-.594.234-.926.234-.266 0-.496-.043-.723-.137-.207-.09-.406-.223-.578-.406-.164-.176-.301-.41-.391-.687s-.141-.586-.141-.93c0-.332.051-.645.141-.926.09-.266.223-.488.383-.664s.352-.312.566-.41c.227-.09.465-.129.727-.129zm-29.09.051h1.211c.445 0 .75.031.902.086a.83.83 0 0 1 .379.289c.09.125.141.273.141.441 0 .219-.059.391-.176.504-.109.133-.281.207-.496.262a3.68 3.68 0 0 1-.789.082h-1.172zm0 0" fill="rgb(60%,60%,60%)"/><path d="M25.953 57.266v1.008h99.824v-1.008zm0 11.93v1.012h99.824v-1.012zM.59 55.484v5.887c3.234-1.266 7.555-2.531 12.988-3.812 3.484-.816 6.852-1.508 10.105-2.074zm23.922 1.258c-3.152.438-5.98.957-8.469 1.539-2.848.672-5.141 1.352-6.875 2.043-1.738.676-2.543 1.227-2.43 1.645.102.375.828.547 2.184.508.699-.004 1.828-.078 3.395-.207l6.074-.566c2.457-.238 4.492-.398 6.121-.488zm-7.004 7.461c-.695.008-1.656.059-2.887.145l-4.59.387-7.078.656-2.363.148v4.77c3.727-.508 7.098-1.121 10.102-1.824 2.953-.695 5.223-1.371 6.816-2.027s2.344-1.207 2.219-1.66c-.102-.387-.852-.582-2.219-.594zm7.004 1.676c-3.273 1.203-7.336 2.383-12.195 3.523C7.797 70.465 3.891 71.23.59 71.699v.039h23.922zm-22.922 1c.012-.004.02 0 .031 0 .063 0 .129.02.172.066.063.059.078.16.043.238-.043.09-.152.141-.254.125-.117-.016-.207-.121-.195-.23.004-.102.094-.187.203-.199zm.848 0c.051 0 .102.012.141.051.078.055.105.16.07.242-.035.09-.129.152-.23.145-.086-.004-.168-.055-.203-.125-.047-.098-.008-.23.094-.281a.28.28 0 0 1 .129-.031zm-.816 1.406a.23.23 0 0 1 .223.152c.031.094-.02.219-.121.258-.098.051-.23.02-.293-.07s-.059-.215.031-.285c.043-.035.102-.059.16-.055zm.816 0c.078-.004.148.039.191.102.063.094.039.227-.059.285-.094.07-.246.059-.32-.031a.22.22 0 0 1 .023-.312c.043-.027.105-.047.164-.043zm0 0" fill="rgb(50.196078%,61.176471%,78.431373%)"/></svg>

```

File: icons/subversion/subversion-original.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...3.004 9.426.547 2.129 4.438 3.098 11.691 2.898 3.73-.07 9.777-.48 18.168-1.223l32.488-3.219c13.145-1.355 24.031-2.273 32.758-2.809zM90.527 66.75c-3.73.047-8.871.332-15.449.816-6.602.492-14.77 1.207-24.559 2.219L12.641 73.5c-4.816.406-8.996.66-12.641.863v27.23c19.938-2.914 37.973-6.391 54.055-10.422 15.797-3.957 27.941-7.82 36.473-11.555 8.547-3.734 12.535-6.902 11.871-9.469-.555-2.215-4.559-3.34-11.871-3.398zM128 76.309c-17.52 6.863-39.266 13.598-65.246 20.121C38.566 102.492 17.672 106.832 0 109.523v.227h128zM5.348 82.02c.063-.008.113 0 .18 0 .332.012.684.113.906.363a1.25 1.25 0 0 1-1.133 2.082c-.617-.09-1.094-.695-1.043-1.312.02-.582.512-1.062 1.09-1.133zm4.531 0c.289.012.555.07.77.273a1.25 1.25 0 0 1 .363 1.402c-.187.496-.691.863-1.227.816-.457-.02-.906-.305-1.086-.727-.25-.547-.047-1.285.5-1.586.203-.121.441-.184.68-.18zm-4.352 8.02c.539-.016 1.023.363 1.18.859.164.555-.098 1.254-.637 1.496-.539.27-1.246.102-1.586-.406-.344-.496-.297-1.242.184-1.633.234-.207.547-.324.859-.316zm4.352 0c.43-.023.809.25 1.039.59a1.23 1.23 0 0 1-.316 1.629c-.512.391-1.312.324-1.723-.18-.422-.504-.391-1.359.137-1.766.23-.191.563-.281.863-.273zm0 0" fill="rgb(50.196078%,61.176471%,78.431373%)"/></svg>
```
Comment (Panquesito7) [inline icons/subversion/subversion-original.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" xmlns:v="https://vecta.io/nano"><path d="M0 17v33.574c17.301-7.203 40.414-14.441 69.504-21.75C88.145 24.152 106.16 20.223 123.559 17zm128 7.16c-16.867 2.504-31.996 5.457-45.309 8.789-15.242 3.828-27.512 7.699-36.793 11.645-9.293 3.875-13.609 7.008-13.004 9.426.547 2.129 4.438 3.098 11.691 2.898 3.73-.07 9.777-.48 18.168-1.223l32.488-3.219c13.145-1.355 24.031-2.273 32.758-2.809zM90.527 66.75c-3.73.047-8.871.332-15.449.816-6.602.492-14.77 1.207-24.559 2.219L12.641 73.5c-4.816.406-8.996.66-12.641.863v27.23c19.938-2.914 37.973-6.391 54.055-10.422 15.797-3.957 27.941-7.82 36.473-11.555 8.547-3.734 12.535-6.902 11.871-9.469-.555-2.215-4.559-3.34-11.871-3.398zM128 76.309c-17.52 6.863-39.266 13.598-65.246 20.121C38.566 102.492 17.672 106.832 0 109.523v.227h128zM5.348 82.02c.063-.008.113 0 .18 0 .332.012.684.113.906.363a1.25 1.25 0 0 1-1.133 2.082c-.617-.09-1.094-.695-1.043-1.312.02-.582.512-1.062 1.09-1.133zm4.531 0c.289.012.555.07.77.273a1.25 1.25 0 0 1 .363 1.402c-.187.496-.691.863-1.227.816-.457-.02-.906-.305-1.086-.727-.25-.547-.047-1.285.5-1.586.203-.121.441-.184.68-.18zm-4.352 8.02c.539-.016 1.023.363 1.18.859.164.555-.098 1.254-.637 1.496-.539.27-1.246.102-1.586-.406-.344-.496-.297-1.242.184-1.633.234-.207.547-.324.859-.316zm4.352 0c.43-.023.809.25 1.039.59a1.23 1.23 0 0 1-.316 1.629c-.512.391-1.312.324-1.723-.18-.422-.504-.391-1.359.137-1.766.23-.191.563-.281.863-.273zm0 0" fill="rgb(50.196078%,61.176471%,78.431373%)"/></svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: devicon.json [hunk 5]
```diff
@@ -3511,35 +3541,33 @@
...
-          "font": ["plain"]
-        },
-        "color": "#cc1e4c",
-        "aliases": []
+        "color": "#809cc8",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            }
```
Comment (Thomas-Boi) [inline devicon.json:3565]: Just need to add a `,` on line 3566
```
            {
                "base": "original",
                "alias": "plain"
            },
```

Comment (Thomas-Boi) [inline devicon.json:3565]: Just need to add a , on line 3566
```suggestion
            },
```

File: devicon.json [hunk 6]
```diff
@@ -3511,35 +3541,33 @@
...
-        "aliases": []
+        "color": "#809cc8",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            },
+            {
+                "base": "original-wordmark",
+                "alias": "original-wordmark"
```
Comment (Thomas-Boi) [inline devicon.json:3568]: Just need to change this to `plain-wordmark`. In the future, you don't have to worry about `aliases`. We are planning to phase it out eventually.
```suggestion
                "alias": "plain-wordmark"
```

File: devicon.json [hunk 7]
```diff
@@ -3623,35 +3653,33 @@
...
-                "original-wordmark",
-                "plain",
-                "plain-wordmark"
+                "original-wordmark"
             ],
             "font": [
-                "plain",
-                "plain-wordmark"
+                "original",
+                "original-wordmark"
```
Comment (Thomas-Boi) [inline devicon.json:3669]: My bad. The `original-wordmark` doesn't fit being an icon.
```suggestion
                "original"
```

File: devicon.json [hunk 8]
```diff
@@ -3623,35 +3653,33 @@
...
+        "color": "#809cc8",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            },
+            {
+                "base": "original-wordmark",
+                "alias": "plain-wordmark"
+            }
```
Comment (Thomas-Boi) [inline devicon.json:3681]: Since `wordmark` version doesn't fit being an icon, we don't need this.
```suggestion
            }
```

**Review by Thomas-Boi — COMMENTED** [review]
I was wrong about the `wordmark` version being icon friendly. I'll remove it now.

**Review by Thomas-Boi — APPROVED** [review]
The icon is great now. Thank you for your work @MichielPater. Looking forward to your other PRs.

**Pull request merged**